<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TomcatWindow</class>
 <widget class="QWidget" name="TomcatWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>599</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>JTFF DTC -Tomcat</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QListWidget" name="WaypointstList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>300</width>
     <height>631</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="WaypointsLabel">
   <property name="geometry">
    <rect>
     <x>85</x>
     <y>10</y>
     <width>150</width>
     <height>25</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>Waypoints</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="NavgridLabel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>13</y>
     <width>70</width>
     <height>21</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="locale">
    <locale language="English" country="UnitedStates"/>
   </property>
   <property name="text">
    <string>Navgrid</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>0</y>
     <width>20</width>
     <height>721</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="NG_Enable">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>15</y>
     <width>70</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Enable</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NG_LatLong">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>60</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QSpinBox" name="NG_Sectors">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>210</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::PlusMinus</enum>
   </property>
   <property name="accelerated">
    <bool>false</bool>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>6</number>
   </property>
   <property name="stepType">
    <enum>QAbstractSpinBox::DefaultStepType</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>241</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="ImportButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>270</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Import from file</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExportButtonToFile">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>390</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Export to file</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExportButtonToAC">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>450</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Export to aircraft</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AddWptButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>680</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RemoveWptButton">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>680</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>348</x>
     <y>65</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Lat/Long</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>110</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Bearing</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>160</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Width</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>210</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Sectors</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="NG_Width">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>160</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::PlusMinus</enum>
   </property>
   <property name="accelerated">
    <bool>false</bool>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
   <property name="stepType">
    <enum>QAbstractSpinBox::DefaultStepType</enum>
   </property>
  </widget>
  <widget class="QSpinBox" name="NG_Bearing">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>110</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::PlusMinus</enum>
   </property>
   <property name="accelerated">
    <bool>false</bool>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>360</number>
   </property>
   <property name="stepType">
    <enum>QAbstractSpinBox::DefaultStepType</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="ImportCFButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>330</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Import from CF</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ModityWptButton">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>680</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>M</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
